docs/wordpress.md

```diff
@@ -93,7 +93,7 @@ database containers. If you're using [Docker Machine](https://docs.docker.com/ma
 
 ## More Compose documentation
 
-- [User guide](/)
+- [User guide](index.md)
```

Comment:
I see 3 other files that use `[User guide](/)` (rails.md, extends.md, install.md).  I think it should be changed in those as well?

Did you validation miss these other files?